pixart.js

```diff
@@ -1 +1,45 @@
 
+var form = document.querySelector('#form')
+var button = document.querySelector('#set-color')
+var field = document.querySelector('#color-field')
+var brush = document.querySelector('.brush')
+
+
+button.addEventListener("click", function (event){
+  event.preventDefault();
+  console.log(brush.style.background)
+
+  brush.style.background = field.value
+  console.log(brush.style.background)
+
+
+})
+
+var divs = []
+
+for ( i = 0; i < 8000; i ++){
+  var div = document.createElement("div");
+  div.setAttribute("class", "square");
+  document.body.appendChild(div);
+  divs.push(div);
+
+};
+
+for (i = 0; i < divs.length;  i++){
+  divs[i].addEventListener("mouseover", function (event){
+    // event.preventDefault();
+    this.style.background = field.value
+  })
+}
```

Comment:
Good job separating concerns (creating squares, adding behavior to squares).  If you extracted these into methods (which I suggest), it would be harder to reuse the `divs` variable.  How would you work with all the appropriate divs if you didn't save them into that variable as you went along?